src/widgets/Scrollbar.cpp

```diff
@@ -153,6 +153,10 @@ void Scrollbar::setDesiredValue(qreal value, bool animated)
                     0.0001;
                 setCurrentValue(value);
             }
+            else
+            {
+                this->currentValueAnimation_.setEndValue(value);
```

Comment:
We're setting the end value here because we don't want to discard the value.

To see why it might be helpful to log all values here as well.